src/bindings/python/src/pyopenvino/utils/utils.cpp

```diff
@@ -258,6 +258,19 @@ std::map<std::string, ov::Any> properties_to_any_map(const std::map<std::string,
     return properties_to_cpp;
 }
 
+bool is_string_null_terminated(const char* str) {
+    if (str == nullptr) {
+        return false;
+    }
+    while (*str) {
```

Comment:
I think this code isn't correct.
In case `str` isn't null terminated, it'd go further in memory while it won't find 0 somewhere else or even crash.
`while (*str)` is equivalent to `while (*str != '\0')`

Reply:
It looks like `strlen` implementation which not work correctly if there is no null character.
Consider use or wrap `memchr(str, null_char, max_bytes_in_str) != nullptr` to find first occurrence of null character